### DISCUSSION TITLE
websocket native client

### DISCUSSION THREAD

**Discussion by carr123** [nats-io/nats-server#2664] [Q&A] [answered]
hello, is there any doc describing how to access nats-server in a golang(c, python, java ...) client  through websocket ?

**Comment by derekcollison** [comment, marked as answer]
Not yet but in progress.

For now you can access via JS and Go clients and a leafnode connection (server to server).
I have the demo system now supporting it, so if you connect via nats cli or one of the above options with wss:// should work..
